package.json

```diff
@@ -48,6 +48,9 @@
     "pnpm": {
         "overrides": {
             "@codesandbox/sandpack-react@^2.12.1>@codesandbox/sandpack-client": "2.10.0"
+        },
+        "patchedDependencies": {
```

Comment:
What was this for? (Just curious)

Reply:
Long story, but vite tests are not running because of a dependency of @udecode/plate is missing import assertions. There have been attempts to solve this within plate (https://github.com/udecode/plate/pull/2991) but their build process removed the "assert" keyword that was added. For now until this is fixed we patched it internally both in in brand-sdk and in web-app.